airflow/operators/bash_operator.py

```diff
@@ -49,6 +49,15 @@ class BashOperator(BaseOperator):
     :type env: dict
     :param output_encoding: Output encoding of bash command
     :type output_encoding: str
+    
+    On execution of the operator the task will up for retry when exception is raised.
+    However if a command exists with non-zero value Airflow will not recognize
+    it as failure unless explicitly specified in the beggining of the script.
+    Example:
+        bash_command = "python3 script.py '{{ next_execution_date }}'"
+        when executing command exit(1) the task will be marked as success.
+        bash_command = "set -e; python3 script.py '{{ next_execution_date }}'"
+        when executing command  exit(1) the task will be marked as up for retry.
```

Comment:
I suspect these aren't going to render quite right - can you run `make -C docs html` then check the rendering of this (I think it writs to docs/build/html/index.html or similar)

Reply:
Also this so that exit(1) is rendered as code/mono-spaced
```suggestion
        when executing command ``exit(1)`` the task will be marked as up for retry.
```